content/ember-data/v2/evented-api.md

```diff
@@ -0,0 +1,10 @@
+---
+id: evented-api-usage
+title: Evented Api Usage
+until: '4.0.0'
+since: '3.12.0-canary.2'
```

Comment:
Are you sure this should be a canary version?

Reply:
not at all, sorry about that, I just used the latest. Should it be 3.12?

Reply:
No problem! Do you know when the deprecation will be introduced? If not, then write "Upcoming Features"

Reply:
3.12 is correct